spec/controllers/admin/instances_controller_spec.rb

```diff
@@ -34,6 +34,20 @@
 
       expect(response).to have_http_status(200)
     end
+
+    describe 'outdated filter parameters' do
+      it 'redirect to status suspended when just limited is set' do
+        get :index, params: { limited: 1 }
+
+        expect(response).to redirect_to admin_instances_path({ status: :suspended })
+      end
+
+      it 'retains other filters when redirecting if limited is set' do
+        get :index, params: { limited: 1, availability: 'failing' }
+
+        expect(response).to redirect_to admin_instances_path({ status: :suspended, availability: 'failing' })
+      end
+    end
```

Comment:
I don't currently have a test in here for `if limited_federation_mode? && params[:status].present?`, which would require doing a context / around block for `ClimateControl.modify LIMITED_FEDERATION_MODE: 'true' do`, which feels like it may be overkill for a simple filter?